### PR TITLE
fix issue #3879: Make maps showing all layer related with analysis

### PR DESCRIPTION
**Pull request by lucernae**
### What does it fix ?
* Ticket #3879
* Funded by : DFAT
* Description : 
   - Add new InaSAFE option: show only impact layer
   - Add new provenance items: original hazard, exposure, and aggregation layer. This is needed to draw map. Since after analysis, hazard, exposure and aggregation were cleaned up, we can't show it using that layer.

### CODE REVIEW

**Comment by lucernae** [comment]
Show all related layers:
order (top to bottom): aggregation, impact, hazard, exposure
screen shot 2017-02-21 at 01 38 49

Show only impact (with exposure):
screen shot 2017-02-21 at 01 40 39

File: safe/definitions/default_settings.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     'visibleLayersOnlyFlag': True,
     'set_layer_from_title_flag': True,
     'setZoomToImpactFlag': True,
+    'set_show_only_impact_on_report': False,
```
Comment (lucernae) [inline safe/definitions/default_settings.py]: the naming is not consistent but I'm trying to follow python convention.

Comment (ismailsunni) [inline safe/definitions/default_settings.py]: Yeah, +1 for lower case. The other are legacy.

**Comment by timlinux** [comment]
Much better thanks @lucernae  - can you make the tests pass so we can merge?

**Pull request merged**